hrjobcontract/tests/phpunit/CRM/Hrjobcontract/BAO/HRJobContractTest.php

```diff
@@ -207,4 +207,35 @@ public function testLengthOfService() {
     $this->assertEquals($lengthOfServiceYmdExpected, CRM_Hrjobcontract_BAO_HRJobContract::getLengthOfServiceYmd($contactId));
   }
 
+  public function testPermanentlyDeleteAllContracts() {
+    $contact = array('first_name' => 'Timothy', 'last_name' => 'Dalton');
+    $contactId = $this->createContact($contact);
```

Comment:
Please use the Contact Fabricator here. You can do that by first importing/aliasing the class:

```php
use CRM_HRCode_Test_Fabricator_Contact as ContactFabricator;
```

and then you create a contact with:
```php
$contact = ContactFabricator::fabricate()
```